tests/dummy/app/adapters/car.js

```diff
@@ -0,0 +1,23 @@
+import JSONAPIAdapter from 'ember-data/adapters/json-api';
+import { AdapterMixin } from 'ember-custom-actions';
+
+export default JSONAPIAdapter.extend(AdapterMixin, {
+  urlForCustomAction(modelName, id, snapshot, requestType) {
+    let { adapterOptions: { suffix } } = snapshot;
+    suffix = suffix || '';
+
+    if (requestType === 'clean') {
+      let baseUrl = this._buildURL(modelName, id);
+      return `${baseUrl}/custom-clean`;
+    } else if (requestType === 'fix') {
+      return `/custom-cars/${id}/custom-fix${suffix}`;
+    } else if (requestType === 'clean-all') {
+      let baseUrl = this._buildURL(modelName, id);
```

Comment:
(restored comment)  
You used this._buildURL in your README but I'm not sure if calling ember private methods is the best solution.

Reply:
Yes, that's what I meant. This is an ember-data private to build a URL. We definitely shouldn't use it in tests.

Reply:
I changed to `this.buildURL` (public)